bash_completion

```diff
@@ -2237,7 +2237,7 @@ _comp_compgen_dvd_devices()
 # @since 2.12
 _comp_compgen_terms()
 {
-    _comp_compgen -a split -- "$({
+    _comp_compgen_split -- "$({
```

Comment:
This will affect also the API of the old `_terms`. I suppose it would be prudent to follow what's been done e.g. with `_usb_ids` and `_pci_ids` etc.

Reply:
added 19ef4ae5 and squashed into a03da8ac.